apps/extension/docs/extension-CI-setup.md

```
@@ -0,0 +1,209 @@
+
+- ✅ Lints and type-checks the extension code
+- 🔨 Builds the extension (`pnpm run build`)
+- 📦 Creates a clean package (`pnpm run package`)
+- 🧪 Runs tests with VS Code test framework
+- 📋 Creates a test VSIX package to verify packaging works
+- 💾 Uploads build artifacts for inspection
+
+### 2. Version & Publish (`version.yml`)
+
+**Triggers:**
+- Push to `main` branch
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)